index.html

```diff
@@ -918,15 +918,21 @@
     </section>
     <section> <h3>Payment scenarios</h3>
       <p>
-        The user buys goods in a store, and payments options include NFC.
+        Payment scenarios with Web NFC generally do not refer to supporting
+        the payment process itself, but recording the fact of payment with a web
```

Comment:
@zolkis please update the PR if this works for you, @kenchris approved the proposed text.